packages/types/src/schemas/common.ts

```diff
@@ -153,9 +153,18 @@ export const webcontainerSchema = commandsSchema.extend({
   i18n: i18nSchema.optional(),
 });
 
+export const editPageLinkSchema = z.union([
+  // pattern for creating the URL
+  z.string(),
+
+  // `false` for disabling the edit link
+  z.boolean(),
+]);
+
 export const baseSchema = webcontainerSchema.extend({
   title: z.string(),
   slug: z.string().optional(),
+  editPageLink: editPageLinkSchema.optional(),
```

Comment:
This should be set inside the (incorrectly named now) `webcontainerSchema` otherwise you can't set the property at the tutorial level which is a bit annoying.